lms/djangoapps/grades/subsection_grade_factory.py

```diff
@@ -104,6 +105,13 @@ def update(self, subsection, only_if_higher=None, score_deleted=False, force_upd
             )
             self._update_saved_subsection_grade(subsection.location, grade_model)
 
+            COURSE_ASSESSMENT_GRADE_CHANGED.send_robust(
```

Comment:
It would be good to update the unit test for this `update()` method to assert that this signal is invoked.

Reply:
Point taken, unit tests coming! 🙂